anyway/models.py

```diff
@@ -866,6 +866,12 @@ class NewsFlash(Base):
     street2_hebrew = Column(Text(), nullable=True)
     non_urban_intersection_hebrew = Column(Text(), nullable=True)
     road_segment_name = Column(Text(), nullable=True)
+    newsflash_location_qualification = Column(
+        Integer(),
+        nullable=False,
+        server_default=text(f"{NewsflashLocationQualification.NOT_VERIFIED.value}"),
```

Comment:
Does this mean the values for existing records, and new records where the field is not explicitly set will be set to NOT_VERIFIED?